src/components/vr_cooperation/include/vr_cooperation/commands/base_command_request.h

```diff
@@ -51,7 +51,7 @@ namespace commands {
  * @brief Base command class for requests
  */
 class BaseCommandRequest : public Command,
-  public event_engine::EventObserver<application_manager::MessagePtr, std::string> {
+  public event_engine::EventObserver<application_manager::MessagePtr, int> {
  public:
```

Comment:
@thinhts it uses for requests to mobile. Separated Base command will be implemented for HMI command factory and will be used for requests to HMI. Here was updated type cause we don't use name of function id in string type as it was in HMI json messages.